programs/zstdless.1

```diff
@@ -1,9 +1,14 @@
-.TH "ZSTDLESS" "1" "April 2022" "zstd 1.5.2" "User Commands"
+.
+.TH "ZSTDLESS" "1" "August 2022" "zstd 1.5.3" "User Commands"
```

Comment:
@terrelln Is it normal that ronn-ng added whitespace to the man files?

Reply:
I deleted all the man pages and re-ran `make man`. Note that the current zstd man pages were created with https://github.com/rtomayko/ronn NOT the newer, maintained one https://github.com/apjanke/ronn-ng. The only one on AUR is `ronn-ng` but the output that is generated is very different and didn't want to blow up the diff. I think switching may be an option but expect a big diff as the `.` in new lines is not there.